hack/tools/release/weekly/main.go

```diff
@@ -169,9 +169,24 @@ func run() int {
 		merges[key] = append(merges[key], formatMerge(body, prNumber))
 	}
 
+	// fetch the current branch
+	out, err = exec.Command("git", "rev-parse", "--abbrev-ref", "HEAD").CombinedOutput()
+	if err != nil {
+		fmt.Println("Error")
```

Comment:
this is a bit confusing, i think we can consolidate the `"Error"` string to print in the same line as `err`

Reply:
I have followed the same pattern which is there in line no: 105 after executing any command.

Reply:
ah cool, if this is a pattern that works for the comms team it works for me 👍